src/Tribe/Rewrite.php

```diff
@@ -272,10 +282,7 @@ public function get_bases( $method = 'regex' ) {
 		$bases = apply_filters( 'tribe_events_rewrite_base_slugs', $default_bases );
 
 		// Remove duplicates (no need to have 'month' twice if no translations are in effect, etc)
-		$bases = array_map( 'array_unique', $bases );
-
-		// By default we always have `en_US` to avoid 404 with older URLs
-		$languages = apply_filters( 'tribe_events_rewrite_i18n_languages', array_unique( array( 'en_US', get_locale() ) ) );
+		$bases = $unfiltered_bases = array_map( 'array_unique', $bases );
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 8323c81bdeb2f08d9737935b17d8a8bb</sub>